### PR TITLE
Taskgraph

### PR DESCRIPTION
**Pull request by escapewindow**
Overall a win.
We do lose coveralls atm.

### CODE REVIEW

File: taskcluster/ci/docker-image/kind.yml [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+
+jobs:
+    python3.10:
+        definition: python
+        args:
+            PYTHON_VERSION: "3.10"
+    python3.9:
+        definition: python
+        args:
+            PYTHON_VERSION: "3.9.7"
```
Comment (hneiva) [inline taskcluster/ci/docker-image/kind.yml:21]: 🤔 any reason why only 3.9 has minor version specified?

Comment (hneiva) [reply taskcluster/ci/docker-image/kind.yml:21]: minor nit: reorder desc or asc.

**Review by hneiva — APPROVED** [review]
LGTM

File: taskcluster/ci/docker-image/kind.yml (hunk 1 above)
Comment (escapewindow) [reply taskcluster/ci/docker-image/kind.yml:21]: Leftover from https://github.com/mozilla-releng/scriptworker-scripts/pull/428 :) I think we had a 3.9 watershed for winsign deps, osslsigncode maybe. Not applicable here, so I can remove.

**Comment by escapewindow** [comment]
Thanks!

**Pull request merged**